invoices/test_helpers.go

```diff
@@ -0,0 +1,195 @@
+package invoices
+
+import (
+	"encoding/hex"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/btcsuite/btcd/btcec"
+	"github.com/btcsuite/btcd/chaincfg"
+	"github.com/lightningnetwork/lnd/channeldb"
+	"github.com/lightningnetwork/lnd/lntypes"
+	"github.com/lightningnetwork/lnd/lnwire"
+	"github.com/lightningnetwork/lnd/record"
+	"github.com/lightningnetwork/lnd/zpay32"
+)
+
+type mockPayload struct {
```

Comment:
these structs are now intermingled w/ the package instead only being compiled with the tests, is that intentional? not a blocker

Reply:
Thanks for pointing out, I somehow forgot Go does this. Renamed `test_helper.go` to `helper_test.go` to avoid this.